### ISSUE TITLE
MPR AXIAL image order is reversed

### ISSUE THREAD

**Issue by liangaili** [nroduit/Weasis#569]
### Describe the bug. What happened?

Hi,

I am using weasis 4.4.0 with Windows 11.

Here's what I do:

1. open a dicom series
2. open MPR 

<img width="186" alt="1" src="https://github.com/nroduit/Weasis/assets/2921944/8857fcfa-c970-4103-96fc-5eb0b6bf0023">

The slice number of a sequence in MPR mode seems reversed:
<img width="159" alt="2" src="https://github.com/nroduit/Weasis/assets/2921944/ef19060f-0e34-4448-bbed-983e1433c97d">

I am not sure if this is a bug or by design.

### What version of Weasis are you running?

4.4.0

### On which system the problem occurs?

Windows

**Comment by nroduit** [comment]
Yes, it is an issue. The sorted order is reversed for all the directions.

**Comment by nroduit** [comment]
After analysis, this is not a bug but simply a sorting according to Anatomical Direction that is forced in MPR mode.

Anatomical Direction is defined in [DICOM](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.6.2.html#sect_C.7.6.2.1.1)
`the x-axis is increasing to the left hand side of the patient. The y-axis is increasing to the posterior side of the patient. The z-axis is increasing toward the head of the patient.`

**Comment by nroduit** [comment]
We're closing this issue due to inactivity over the past several months. If the problem persists or if you have any further information to provide, please feel free to reopen this issue or create a new one.
We appreciate your contributions and feedback!
